Adyen/Assets/Generated/ADYLocalizationKey.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+// This file is autogenerated. Please do not modify it.
+
+/// :nodoc:
+public struct ADYLocalizationKey {
```

Comment:
why not an enum? why a struct?

Reply:
No preference really. I thought you were against big enums :)